### ISSUE TITLE
Error regarding squeeze.c during setup

### ISSUE THREAD

**Issue by Arno-Enslin** [google/zopfli#115]
Regarding https://github.com/anthrotype/py-zopfli

On 03-26-2016 I was able to integrate Zopfli into the FontTools. But you have something changed in [squeeze.c](https://github.com/google/zopfli/blob/master/src/zopfli/squeeze.c). And now I am getting this error message, when I try to integrate Zopfli:

```
squeeze.c
zopfli/src/zopfli/squeeze.c(199) : error C2059: syntax error : 'type'
zopfli/src/zopfli/squeeze.c(248) : warning C4305: '=' : truncation from 'double'
 to 'float'
zopfli/src/zopfli/squeeze.c(269) : warning C4244: '=' : conversion from 'double'
 to 'float', possible loss of data
zopfli/src/zopfli/squeeze.c(286) : warning C4244: '=' : conversion from 'double'
 to 'float', possible loss of data
zopfli/src/zopfli/squeeze.c(300) : warning C4267: 'function' : conversion from '
size_t' to 'unsigned int', possible loss of data
zopfli/src/zopfli/squeeze.c(304) : warning C4244: '=' : conversion from 'double'
 to 'float', possible loss of data
zopfli/src/zopfli/squeeze.c(305) : warning C4267: '=' : conversion from 'size_t'
 to 'unsigned short', possible loss of data
error: command 'C:\\Users\\XX\\AppData\\Local\\Programs\\Common\\Microsoft\\Visu
al C++ for Python\\9.0\\VC\\Bin\\amd64\\cl.exe' failed with exit status 2
```

**Comment by JayXon** [comment]
Does #107 fix your problem?

**Comment by anthrotype** [comment]
It does, but maybe it's preferable keep the source code as is, and pass instead the `/Za` flag which disables non-ANSI C language extensions like the `min` macro. This is consistent with the current zopfli Makefile, which has a similar `-ansi` flag.

**Comment by Jesin** [comment]
Should this issue be closed now that #107 has been merged?
